client/src/components/ListGroup/ListGroup.scss

```diff
@@ -55,11 +56,16 @@
 
 .list-group-item__thumbnail {
   width: 64px;
-  height: 64px;
+  height: auto;
   display: block;
   background: $gray-lighter;
   float: left;
-  margin: -12px 12px 0 #{0 - $grid-gutter-width / 2};
+  margin: -12px 12px -12px #{0 - $grid-gutter-width / 2};
+
+  > img {
+    max-height: 65px;
+    height: 100%;
```

Comment:
yeah, my counter argument is that stretching an image is misrepresentation of what the image really is and we do not stretch the preview in the AssetAdmin Gallery either.